packages/halo/src/react/__specs__/storyshots.spec.tsx

```diff
@@ -4,12 +4,12 @@ import initStoryshots, {
   snapshotWithOptions
 } from '@storybook/addon-storyshots'
 
-jest.mock('../__stories__/focusable.js', () => props => <div {...props} />)
-
-const createNodeMock = () => document.createElement('div')
+jest.mock(
+  '../__stories__/focusable',
+  () => (props: React.HTMLAttributes<HTMLDivElement>) => <div {...props} />
+)
 
 initStoryshots({
   configPath: path.resolve(__dirname, '../../../.storybook'),
-  framework: 'react',
-  test: snapshotWithOptions({ createNodeMock })
+  framework: 'react'
 })
```

Comment:
@jaketrent thanks i never know how to get this to work.

also we we not need the node mock anymore?

Reply:
I guess not! 🤷   I looked at the type of `snapshotWithOptions`, and it said it didn't even take that option, `createNodeMock`.  I removed it, tests ran and snapshots were the same.